src/abstract.jl

```diff
@@ -204,3 +216,43 @@ end
 function newids(man::AbstractCutPruner, n::Int)
     (man.id+1):(man.id += n)
 end
+
+
+"""
+Check redundant cuts. Return index of redundant cuts in `Anew`.
+
+$(SIGNATURES)
+
+"""
+function checkredundancy{T}(A::AbstractMatrix{T}, b::AbstractVector{T},
+                            Anew::AbstractMatrix{T}, bnew::AbstractVector{T})
+    # index of redundants cuts
+    redundants = Int[]
+    # number of new lines
+    nnew = size(Anew, 1)
+
+    for kk in 1:nnew
+        λ = @view Anew[kk, :]
+        chk, indk = isinside(A, λ)
+        if chk && (bnew[kk] <= b[indk])
+            push!(redundants, kk)
+        end
+    end
+
+    redundants
+end
+
+
+"""Check if `λ` is a line of matrix `A`."""
+function isinside{T}(A::AbstractMatrix{T}, λ::AbstractVector{T})
+    nlines = size(A, 1)
+
+    check = false
+    k = 0
+    while ~check && k < nlines
+        k += 1
+        check = (A[k, :] == λ)
```

Comment:
I just saw the FAST interface. In fact, I think that is a good idea to use the infinity norm and set a tolerance `eps` in the definition of the pruner.